content/reference/arvo/clay/scry.md

```diff
@@ -141,36 +312,42 @@ Here we scry the [dome:clay](/reference/arvo/clay/data-types#dome-desk-data) for
 ]
 ```
 
-### %blob - File blob.
+---
+
+### `%blob` - File blob
 
-This will return the [blob:clay](/reference/arvo/clay/data-types#blob-clay-data) of some file. It takes a [lobe:clay](/reference/arvo/clay/data-types#lobe-clay-data-reference).
+This will return the [page:clay](/reference/arvo/clay/data-types#pageclay) of
+some file. It takes a [lobe:clay](/reference/arvo/clay/data-types#lobeclay).
 
 Example:
 
-Here we grab the `lobe:clay` of `/gen/hood/hi/hoon` with a `%y` scry, then use it to do a `%s` scry for the `blob:clay` of the file.
+Here we grab the `lobe:clay` of `/gen/hood/hi/hoon` with a `%y` scry, then use
+it to do a `%s` scry for the `blob:clay` of the file.
 
 ```
 > =/  =arch  .^(arch %cy %/gen/hood/hi/hoon)
   ?~  fil.arch
     ~
-  .^(blob:clay %cs %/blob/(scot %uv u.fil.arch))
-[ %direct
-  p=0vp.k7nsm.qkdr3.t17rp.33bae.8gajg.v27gi.40itr.2u9qa.nppbt.7k255
+  .^(page:clay %cs %/blob/(scot %uv u.fil.arch))
+[ p=%hoon
     q
-  [ p=%hoon
-      q
-3.548.750.706.400.251.607.252.023.288.575.526.190.856.734.474.077.821.289.791.377.301.707.878.697.553.411.219.689.905.949.957.893.633.811.025.757.107.990.477.902.858.170.125.439.223.250.551.937.540.468.638.902.955.378.837.954.792.031.592.462.617.422.136.386.332.469.076.584.061.249.923.938.374.214.925.312.954.606.277.212.923.859.309.330.556.730.410.200.952.056.760.727.611.447.500.996.168.035.027.753.417.869.213.425.113.257.514.474.700.810.203.348.784.547.006.707.150.406.298.809.062.567.217.447.347.357.039.994.339.342.906
-  ]
+  3.548.750.706.400.251.607.252.023.288.575.526.190.856.734.474.077.821.289.791.377.301.707.878.697.553.411.219.689.905.949.957.893.633.811.025.757.107.990.477.902.858.170.125.439.223.250.551.937.540.468.638.902.955.378.837.954.792.031.592.462.617.422.136.386.332.469.076.584.061.249.923.938.374.214.925.312.954.606.277.212.923.859.309.330.556.730.410.200.952.056.760.727.611.447.500.996.168.035.027.753.417.869.213.425.113.257.514.474.700.810.203.348.784.547.006.707.150.406.298.809.062.567.217.447.347.357.039.994.339.342.906
 ]
 ```
 
-### %hash - Commit hash.
+---
+
+### `%hash` - Commit hash
 
-This will return the `@uvI` content hash of the specified commit. It takes a `tako:clay`.
+This will return the `@uvI` content hash of the specified commit. It takes a
```

Comment:
Maybe parenthetically (256-bit) since I always have to calculate these from J...